src/Console/Application.php

```diff
@@ -122,7 +122,8 @@ public function getLongVersion(): string
         $version = implode('', [
             parent::getLongVersion(),
             self::VERSION_CODENAME ? sprintf(' <info>%s</info>', self::VERSION_CODENAME) : '', // @phpstan-ignore-line to avoid `Ternary operator condition is always true|false.`
-            ' by <comment>Fabien Potencier</comment> and <comment>Dariusz Ruminski</comment>',
+            ' by <comment>Fabien Potencier</comment> and <comment>Dariusz Ruminski</comment>.',
+            ' PHP runtime: <comment>'.PHP_VERSION.'</comment>',
```

Comment:
can we keep the new line? I would love to add extra info over time, and having everything in one liner would make it super long

```suggestion
            "\n".' PHP runtime: <comment>'.PHP_VERSION.'</comment>',
```